Cubical/Codata/Conat/Base.agda

```diff
@@ -0,0 +1,54 @@
+{- Conatural numbers (Tesla Ice Zhang, Feb. 2019)
+
+This file defines:
+
+- A coinductive natural number representation which is dual to
+  the inductive version (zero | suc Nat → Nat) of natural numbers.
+
+- Trivial operations (succ, pred) and the pattern synonyms on conaturals.
+
+While this definition can be seen as a coinductive wrapper of an inductive
+family, another way of definition is to define an inductive family that wraps
```

Comment:
The use of "inductive family" makes it seem like we are using indices, but we are not. Maybe just "inductive type", or possibly something else to that effect.

Reply:
Fixed.